<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InOutPage</class>
 <widget class="QWizardPage" name="InOutPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="IOWidget" name="m_inputWidget" native="true"/>
   </item>
   <item>
    <widget class="QPushButton" name="m_addInput">
     <property name="text">
      <string>Add Input</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="IOWidget" name="m_outputWidget" native="true"/>
   </item>
   <item>
    <widget class="QPushButton" name="m_addOutput">
     <property name="text">
      <string>Add Output</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IOWidget</class>
   <extends>QWidget</extends>
   <header>iowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
